test/e2e/DCAHub/happy-path.spec.ts

```diff
@@ -94,8 +94,6 @@ contract('DCAHub', () => {
       setSwapRatio(swapRatioAC1);
       setSwapRatio(swapRatioBC1);
 
-      await assertNoSwapsCanBeExecutedNow();
```

Comment:
This will fail now, because we can't call `getNextSwapInfo` with tokens that haven't been deposited at least once